SwrveConversationSDK/Conversation/SwrveContentStarRating.m

```diff
@@ -5,7 +5,7 @@
 #import "SwrveSetup.h"
 
 #define kSwrveKeyStarColor @"star_color"
-#define kSwrveStarRatingHeight 110.0f
+#define kSwrveStarRatingHeight 60.0f
```

Comment:
The size was hyperbolic in the original release, I found that I didn't need to fix to size to such a massive height to be in line with Android